modin/core/dataframe/base/dataframe/utils.py

```diff
@@ -36,6 +42,10 @@ class Axis(Enum):  # noqa: PR01
     CELL_WISE = None
 
 
+AxisInt = Literal[0, 1]
+"""Type for the two possible integer values of an axis argument (0 or 1)."""
+
+
```

Comment:
A lot of the codebase (mostly the query compiler) is written to call dataframe methods with a literal int rather than the `Axis` enum. I think it would be easier to re-wrap the axis with the enum from within dataframe methods (as is done now) than to go through and fix every instance where relevant dataframe methods are called to use the enum instead.